cdap-ui/app/cdap/components/Market/AllTab/index.js

```diff
@@ -29,12 +29,17 @@ export default class AllTabContents extends Component {
     this.state = {
       searchStr: '',
       entities: this.getFilteredEntities(),
+      filterEntites: this.getFilteredEntities(),
```

Comment:
Can we do this once and store it in a local variable and use them here instead of calling it twice?

```
const filteredEntities = this.getFilteredEntities();
this.state = {
  searchStr: '',
  entities: filteredEntities,
  filterEntities: filteredEntities,
  loading: MarketStore.getState().loading,
  isError: MarketStore.getState().isError,
};
```

Reply:
done